.github/workflows/sync-registry.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs every midnight
```

Comment:
semantically, i would keep this as a `workflow_dispatch`. These images will not be changing every midnight, so no need to run this on a cron.